.goreleaser.yml

```
@@ -0,0 +1,30 @@
+builds:
+  - env:
+      - CGO_ENABLED=0
+    ldflags:
+      - "-s -w -X github.com/lunarway/strong-duckling/cmd.version={{.Version}} -X github.com/lunarway/strong-duckling/cmd.commit={{.Commit}}"
```

We only have the version field available right now.
```suggestion
      - "-s -w -X github.com/lunarway/strong-duckling/cmd.version={{.Version}}"
```